task.py

```diff
@@ -107,3 +107,12 @@ def run(self, duration: int):
     @abstractmethod
     def stop(self):
         raise NotImplementedError('Must implement stop()')
+    
+    """
+    output() should be called to store the results of this task in a machine readable format (i.e. json) in the log location specified by the user,
+    as well as print any required info/debug to the console. The results should be formatted such that other modules can easily consume the output, such
+    as a module to determine the success/failure/performance of a given run.
+    """
+    @abstractmethod
+    def output(self):
```

Comment:
It would be nice to have a way to output the machine readable to a specific file.

Reply:
Currently all the machine readable logs will be output to the log path specified by the user. i.e.

```
# ls /tmp/ft-logs/2023-08-22-11-09-40/
2-SRIOV_TO_POD_IP_TO_SRIOV-DIFF_NODE.json  3-SRIOV_TO_POD_IP_TO_HOSTBACKED-SAME_NODE.json  4-SRIOV_TO_POD_IP_TO_HOSTBACKED-DIFF_NODE.json
```

I was thinking we would always output the machine readable by default since it is a superset of the human readable output and will be more persistent than terminal output.